src/exchange.js

```diff
@@ -69,32 +82,26 @@ class Exchange {
     return this.getBuyQuote(-amount, baseCurrency, quoteCurrency);
   }
 
-  updateList (list, items, ListClass) {
-    var item;
-    for (var i = 0; i < items.length; i++) {
-      item = undefined;
-      for (var k = 0; k < list.length; k++) {
-        var itemId = Helpers.isNumber(items[i].id) ? items[i].id : items[i].id.toLowerCase();
-        if (list[k]._id === itemId) {
-          item = list[k];
-          item.debug = this.debug;
-          item.set.bind(item)(items[i]);
-        }
-      }
-      if (item === undefined) {
-        item = new ListClass(items[i], this._api, this.delegate, this);
-        item.debug = this.debug;
-        list.push(item);
-      }
-    }
-  }
-
   getTrades () {
     var save = () => {
       return this.delegate.save.bind(this.delegate)().then(() => this._trades);
     };
-    var update = (trades) => {
-      this.updateList(this._trades, trades, this._TradeClass);
+    var update = (tradeObjects) => {
+      for (let tradeObj of tradeObjects) {
+        let id = this._TradeClass.idFromAPI(tradeObj);
```

Comment:
It doesn't look like `idFromAPI` is implemented in src/Trade, is this Unocoin specific?